releasenotes/notes/fix-type-angles-euler-decompose-233e5cee7205ed03.yaml

```diff
@@ -0,0 +1,12 @@
+---
+fixes:
+  - |
+    Fixed an issue with the :class:`~.OneQubitEulerDecomposer` class's methods
+    :meth:`~.OneQubitEulerDecomposer.angles` and :meth:`~.OneQubitEulerDecomposer.angles_and_phase`
+    would error if the input matrix was of a dtype other than ``complex``/``np.cdouble``. While
+    only a complex matrix can be unitary if the imaginary component is 0 for
+    all elements it could be represented by a different data type. In earlier
```

Comment:
I'm not sure what the sentence is trying to say here - a unitary matrix can certainly be complex-valued (e.g. Pauli Y).

Reply:
Removed in: https://github.com/Qiskit/qiskit-terra/pull/9828/commits/448470ac1620c38cb543704aa1e92d2673b483fd